### DISCUSSION TITLE
Defaulting named workspaces to a monitor

### DISCUSSION THREAD

**Discussion by Xeals-Senpai** [hyprwm/Hyprland#582] [Q&A]
Hi, title says it all but has this been implemented yet? If so, I'd like to ask for the proper line style.

I have tried the following styles:

- workspace=Monitor#1, name:name#1
- workspace=Monitor#2,name#2
- $name3 = name#3
  workspace=Monitor#3,$name3

**Comment by vaxerski** [comment]
```
workspace=DP-3,name:Hello World
```
works for me.

Please keep in mind, as the wiki states, that you need to have a specific monitor rule set for the monitor you're applying the default workspace to: https://wiki.hyprland.org/Configuring/Advanced-config/#rotating-and-the-default-workspace
